csrankings-c.csv

```diff
@@ -591,7 +591,7 @@ Chris Clack,University College London,http://www0.cs.ucl.ac.uk/people/C.Clack.ht
 Chris Clifton,Purdue University,https://www.cs.purdue.edu/people/clifton,C_cWJIkAAAAJ
 Chris Crispin-Bailey,University of York,https://www-users.cs.york.ac.uk/~chrisb,QQ_dWtcAAAAJ
 Chris De Sa,Cornell University,http://www.cs.cornell.edu/~cdesa,v7EjGHkAAAAJ
-Chris Ding,Chinese University of Hong Kong,https://sds.cuhk.edu.cn/en/teacher/197,q7FfnjgAAAAJ
+Chris Ding,The Chinese University of Hong Kong (Shenzhen),https://sds.cuhk.edu.cn/en/teacher/197,q7FfnjgAAAAJ

```

Comment:
This institution is not in the database, so please remove or add all faculty for that institution and an entry in `country-info.csv`. Thanks.